autopts/pybtp/defs.py

```diff
@@ -158,6 +158,10 @@ def BIT(bit):
 BTP_GAP_CMD_PAIR_V2_FLAG_NONE = 0x00
 BTP_GAP_CMD_PAIR_V2_FLAG_FORCE_PAIR = 0x01
 BTP_GAP_CMD_PAIR_V2 = 0x2a
+BTP_GAP_CMD_BIG_CREATE_SYNC_BCODE_SIZE = 0x10
```

Comment:
The BCode size is not a BTP defined size, nor should it be placed next to the commands IMO. 

Not sure what the convention here is though @sjanc ?